packages/terra-clinical-data-grid/src/terra-dev-site/test/clinical-data-grid/LabelExternalDataGrid.test.jsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
```

Comment:
This test file and the next one were being used for local devsite testing and may not need to be merged to main